src/App.tsx

```diff
@@ -34,6 +35,22 @@ export function getReorderedGoods(
   // To avoid the original array mutation
   const visibleGoods = [...goods];
 
+  if (sortType === SortType.LENGTH) {
+    visibleGoods.sort((g1, g2) => (
+      g1.length - g2.length
+    ));
+  }
+
+  if (sortType === SortType.ALPABET || sortType === SortType.NONE) {
+    visibleGoods.sort((g1, g2) => (
```

Comment:
this solution make sort list in  alphabet by default it's wrong, because we have to do it on click, by default  our list have to be SortType.NONE